exercises/practice/etl/etl.sql

```diff
@@ -0,0 +1,2 @@
+-- Schema: CREATE TABLE "etl" ("input" TEXT, "result" TEXT);
```

Comment:
Did we want to move this metadata from stubs to the `instructions.append`? And/or have it in both places?
@exercism/sqlite

Reply:
It can’t hurt to have it in both places so folks get an idea of what’s expected before starting.

Reply:
Like https://github.com/exercism/sqlite/blob/main/exercises/practice/resistor-color/.docs/instructions.append.md ?

Reply:
Yup.

Reply:
I've added this to all 3 open PRs.